activejob/lib/active_job/test_helper.rb

```diff
@@ -98,6 +98,24 @@ def queue_adapter_for_test
     #     end
     #   end
     #
+    # The number of times a unspecified job is enqueued can be asserted.
+    #
+    #   def test_logging_job
+    #     assert_enqueued_jobs 1, except: HelloJob do
+    #       LoggingJob.perform_later
+    #       HelloJob.perform_later('jeremy')
+    #     end
+    #   end
+    #
+    # The number of times a specified job is enqueued can be asserted, not a job unspecified.
+    #
+    #   def test_logging_job
+    #     assert_enqueued_jobs 1, only: LoggingJob, except: LoggingJob do
```

Comment:
Can you clarify this example? It's unclear to me what this is meant to be testing. It seems like the value would be 0 here? (Or an error)